tokens/lyfe/LYFE-6AB.json

```diff
@@ -0,0 +1,16 @@
+{
+    "asset":"LYFE-6AB",
+    "assetMap":"LYFE",
+    "officialSiteUrl":"https://lyfe.health/",
```

Comment:
Please add the market info according to the updated template: https://github.com/binance-chain/tokens-info/blob/master/PULL_REQUEST_TEMPLATE.md

Reply:
Hi,

I have added the market info.

Thank you